internal/api/auth.go

```diff
@@ -24,11 +29,20 @@ func AuthenticateMiddleware(authn smqauthn.Authentication, domainCheck bool) fun
 				EncodeError(r.Context(), apiutil.ErrBearerToken, w)
 				return
 			}
-
-			resp, err := authn.Authenticate(r.Context(), token)
-			if err != nil {
-				EncodeError(r.Context(), err, w)
-				return
+			var resp smqauthn.Session
+			var err error
```

Comment:
@nyagamunene 
Have you tested this? 
I think code will not reach here if we sent `PAT` in Bearer token

Reply:
It does reach to this code. I have used PAT in authorization and I send the PAT in bearer token